trio/_file_io.py

```diff
@@ -125,18 +291,167 @@ async def aclose(self):
 
         await trio.lowlevel.checkpoint_if_cancelled()
 
+    if TYPE_CHECKING:
+        # fmt: off
```

Comment:
I don't think you ever reenable black formatting?

Reply:
I thought I read that it'd just re-enable after the dedent, but perhaps not.

Reply:
Oh it does. My bad.